eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/SecurityPass.php

```diff
@@ -32,29 +32,34 @@ public function process(ContainerBuilder $container)
         }
 
         $configResolverRef = new Reference('ezpublish.config.resolver');
-        $repositoryReference = new Reference('ezpublish.api.repository');
+        $permissionResolverRef = new Reference('eZ\Publish\API\Repository\PermissionResolver');
+        $userServiceRef = new Reference('eZ\Publish\API\Repository\UserService');
```

Comment:
``` php
use eZ\Publish\API\Repository\UserService;
// ...
$userServiceRef = new Reference(UserService::class);
```
but investigate if those setters are still needed. Maybe we just need a constructor for this class with auto-wiring enabled.